docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -245,6 +240,12 @@ Rule settings:
 
 *`when`*: Condition which must succeed in order to execute the current rule.
 
+ifeval::["{beatname_lc}" == "apm-server"]
+NOTE: When changing APM indices, you have to update the {kibana-ref}/apm-settings-kb.html[Kibana APM settings]
```

Comment:
That's true, but then someone might think "In which cases do I need to update" and I prefer not to explain this here, as it would couple too strong to the default value in Kibana.